client/src/main/java/hudson/plugins/swarm/Client.java

```diff
@@ -29,7 +29,7 @@
     private static final Logger logger = Logger.getLogger(Client.class.getPackage().getName());
 
     private final Options options;
-    private final Thread labelFileWatcherThread = null;
+    //private static final Thread labelFileWatcherThread = null;
```

Comment:
Commented-out code is generally considered bad practice [for a number of reasons](https://kentcdodds.com/blog/please-dont-commit-commented-out-code). This code will always be available in the version history if we need to refer to it at some point in the future, so would you be willing to delete this line completely?

Reply:
thanks a lot for your guiding and reviewing. I have removed the Commented-out code.